src/guessTheNumber.js

```diff
@@ -0,0 +1,33 @@
+let mysteryNumber;
+let docInput;
+
+function setNumber() {
+  docInput =  document.getElementById('answer').value
+  if(!mysteryNumber)
+    mysteryNumber = (Math.floor(Math.random() * 100) + 1).toString()
+}
+function combined() {
+  setNumber()
+  guessTheNumber(docInput, mysteryNumber)
+}
+
+function guessTheNumber(docInput, mysteryNumber) {
+  console.log('mystery number ' + mysteryNumber)
+  let input = docInput
+
+  // let input = document.getElementById('answer').value
+  if((typeof docInput != Number) && (docInput.toLowerCase() == 'exit')) {
+    window.location="/Users/jhallman5/Desktop/Coding/coding-exercises-week-9/pages/home.html" } //why does this always redirect home?
```

Comment:
I'm confused about the comment and the code -- it's asking why it's redirecting home and the link is your home page